control-plane/cni/main.go

```diff
@@ -132,9 +135,11 @@ func (c *Command) cmdAdd(args *skel.CmdArgs) error {
 
 	podNamespace := string(cniArgs.K8S_POD_NAMESPACE)
 	podName := string(cniArgs.K8S_POD_NAME)
+	cniArgsIPTablesCfg := string(cniArgs.IPTABLES_CONFIG)
```

Comment:
Note for reviewers: this arg name was what we landed on in the [RFC NMD-191 (internal link)](https://docs.google.com/document/d/1GTC-j6sIgQ3LtYcuoDxnAcluenKZU4WHybth2ngcoOc/edit) so that it's generic rather than specific to Nomad. But if we think this is likely to have a namespace collision with other CNI args in the ecosystem, we might want this to be HashiCorp-specific. I don't have a good handle on whether ecosystem for k8s cares much about this, so I'll defer to experts on that here. I'll just need to update the equivalent argument name in https://github.com/hashicorp/nomad/pull/20183

Reply:
> But if we think this is likely to have a namespace collision with other CNI args in the ecosystem, we might want this to be HashiCorp-specific.

That's a good callout-- I don't know of any specific other CNI's that take in an IPTABLES_CONFIG arg, but it seems like since it's easy enough to make the name more specific we might as well do it now rather than have the potential to run into collisions? Since that is quite a generic arg name and another CNI could plausibly be configuring iptables.

Also an idea is it could be feature/tproxy specific instead of hashicorp specific and that way company name doesn't have to make it into an argument name.

Reply:
How does `TPROXY_IPTABLES_CONFIG` sound?

Reply:
Both seem fine, maybe `CONSUL_IPTABLES_CONFIG` because we sometimes reserve the consul prefix in other cases like for metadata.